### PR TITLE
Continue redux for state management

### PR DESCRIPTION
**Pull request by BogdanNegura**

### CODE REVIEW

File: src/components/logs/EditLogModal.js [hunk 1]
```diff
@@ -1,16 +1,37 @@
-import React, { useState } from 'react'
+import React, { useState, useEffect } from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
```
Comment (marianzburlea) [inline src/components/logs/EditLogModal.js]: you're not destructuring PropTypes, like you're destructuring { connect } an { useState, useEffect } ////

File: src/actions/logActions.js [hunk 2]
```diff
@@ -116,6 +117,26 @@ export const updateLog = log => async dispatch => {
     }
 }
 
+// Search server logs
+export const searchLogs = (text) => async dispatch => {
```
Comment (marianzburlea) [inline src/actions/logActions.js]: why do you need () around text?

File: src/actions/logActions.js [hunk 3]
```diff
@@ -116,6 +117,26 @@ export const updateLog = log => async dispatch => {
...
+        const res = await fetch(`/logs?q=${text}`)
+        const data = await res.json()
+
+        dispatch({
+            type: SEARCH_LOGS,
+            payload: data
+        })
+    } catch (err) {
+        dispatch({
+            type:LOGS_ERROR,
```
Comment (marianzburlea) [inline src/actions/logActions.js]: missing space

File: src/components/layout/SearchBar.js [hunk 4]
```diff
@@ -1,12 +1,20 @@
-import React from 'react'
+import React, { useRef } from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
```
Comment (marianzburlea) [inline src/components/layout/SearchBar.js]: destrcuture PropTypes

File: src/actions/logActions.js (hunk 2 above)
Comment (BogdanNegura) [reply src/actions/logActions.js]: yeah, it's not necessary, I forgot :)

**Pull request merged**